integration/SimpleCacheClient.test.ts

```diff
@@ -10,38 +9,46 @@ const AUTH_TOKEN = process.env.TEST_AUTH_TOKEN;
 if (!AUTH_TOKEN) {
   throw new Error('Missing required env var TEST_AUTH_TOKEN');
 }
-const INTEGRATION_TEST_CACHE_NAME = process.env.TEST_CACHE_NAME || 'dummy';
+const INTEGRATION_TEST_CACHE_NAME =
+  process.env.TEST_CACHE_NAME || 'integration-test';
```

Comment:
Same here? Might be worth calling this like `js-integration-test-default` or something.